catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -116,11 +117,11 @@
   )
   int pageSize = 1000;
 
-  // DDF-535: remove "Transformer" alias in DDF 3.0
+  // DDF-535: remove "TransformerProperties" alias in DDF 3.0
   @Option(
     name = "--transformer",
     required = false,
-    aliases = {"-t", "Transformer"},
+    aliases = {"-t", "TransformerProperties"},
```

Comment:
Is this just a rename or does that actual behavior of the `-t` option change in this PR?

Reply:
oops, IntelliJ got a little aggressive with a refactor/rename. Fixed.